transportable-udfs-avro/src/test/java/com/linkedin/transport/avro/TestAvroWrapper.java

```diff
@@ -94,10 +99,15 @@ public void testDoubleType() {
   }
 
   @Test
-  public void testStringType() {
+  public void testStringTypeUtf8() {
```

Comment:
Minor: just merge this test with the stringtypetest

Reply:
Thanks, this will be cleaner.